vor-android/mobile/src/main/java/com/futurice/hereandnow/pojo/Toilet.kt

```diff
@@ -1,19 +0,0 @@
-package com.futurice.hereandnow.pojo
-
-import android.content.SharedPreferences
-
-/**
- * Created by Luís Ramalho on 28/12/15.
- * <luis.ramalho@futurice.com>
- */
-class Toilet {
```

Comment:
We could probably remove Kotlin from gradle now that it isn't used. Or was there something else as well?